packages/fxa-auth-server/lib/senders/emails/templates/index.ts

```diff
@@ -1 +1 @@
-export * as cadReminder from './cadReminder';
+export * as cadReminderMJML from './cadReminder';
```

Comment:
Exporting it as `cadReminderMJML` since @vbudhram has included a regex pattern in [this PR](https://github.com/mozilla/fxa/pull/9808/commits/3c77c9039b3c6ed22638be9ff0798e170ac08340) and I thought we could use `MJML` as a pattern to enable feature flagging for the new emails